src/layout/Breadcrumbs.tsx

```diff
@@ -20,7 +19,7 @@ const StyledBreadcrumbs = styled.div`
 
 const Breadcrumbs: React.FC = () => {
   const location = useLocation();
-  const { t } = useTranslation();
+  const { t } = useTranslation('breadcrumbs');
```

Comment:
smart!

angående refaktorering av breadcrumbs: er det vanlig å vise breadcrumbs for en ugyldig sti? 
![Skjermbilde 2019-11-27 kl  11 11 25](https://user-images.githubusercontent.com/6595982/69714434-ad1c2400-1106-11ea-99ba-58d37e7bbf9e.png)

Reply:
Tenker jeg lager en ny PR som deler opp hvilke komponenter som skal ha breadcrumbs og ikke. Små PR er bra PR!